.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
```

Comment:
Please increase max line length to 120 characters (and use across the board for projects).